core/src/main/java/org/infinispan/stream/StreamMarshalling.java

```diff
@@ -113,6 +137,7 @@ public V apply(Map.Entry<K, V> kvEntry) {
       private static final int ENTRY_KEY_FUNCTION = 1;
       private static final int ENTRY_VALUE_FUNCTION = 2;
       private static final int NON_NULL_PREDICATE = 3;
+      private static final int ALWAYS_TRUE_PREDICATE = 4;
```

Comment:
Sure, I can change this over if you think it is a bit more clear (I agree it would be less error prone).